mpt_trie/src/nibbles.rs

```diff
@@ -26,6 +26,11 @@ pub type Nibble = u8;
 /// Used for the internal representation of a sequence of nibbles.
 pub type NibblesIntern = U512;
```

Comment:
Yeah exactly. It's for the 65 `Nibble` edge case.

I don't think I have a Github issue open for this (there was one in Jira), so I'll create one. Also worth having a discussion on the best way to implement this. If we care about performance, using a byte array *might* perform worse. I know `H256` uses a `[u8; 32]`, but `U256`s use `[u64; 4]`. If I had to guess, since `H256` don't support common operators (eg. `+`, `-`), they probably found that working with `u64`s allowed them to implement a lot of operations with 64-bit instructions (vs. doing a bunch of operations to individual bytes). Probably worth talking to @dvdplm about this, as IIRC he worked on `ethereum-types` a good amount.

Reply:
Created an issue for this (#104).